asgi.py

```diff
@@ -0,0 +1,19 @@
+import uvicorn
+
+from src.app.environment import get_settings
+from src.app.log import CONFIG as LOGGING_CONFIG
+
+settings = get_settings()
+
+
+if __name__ == "__main__":
+    server = uvicorn.Server(
+        uvicorn.Config(
+            "src.app.api:app",
+            host=settings.host,
+            port=settings.port,
+            reload=settings.app_reload,
+            log_config=LOGGING_CONFIG,
```

Comment:
https://www.uvicorn.org/settings/#logging

I think this has the same effect as `logging.config.dictConfig`